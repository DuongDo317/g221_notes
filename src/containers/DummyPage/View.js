import React from 'react'
import Foo from './components/Foo'
import Bar from './components/Bar'

export default function View(props) {
    return (
    <>
        <Foo>Foo...</Foo>
        <Bar>Bar...</Bar>
    </>
    )
}
