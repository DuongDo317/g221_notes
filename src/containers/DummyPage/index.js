import React, { Component } from 'react'
import Foo from './components/Foo'
import Bar from './components/Bar'

export default class DummyPage extends Component {
  constructor(props) {
    super(props)
    this.state = {
      foo: 'foo',
      bar: 'bar'
    }
  }

  render() {
    return (
      <>
        <Foo>Foo...</Foo>
        <Bar>Bar...</Bar>
      </>
    )
  }
}
