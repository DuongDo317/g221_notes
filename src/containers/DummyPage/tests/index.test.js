import React from 'react'
import { render } from '@testing-library/react'

import DummyPage from '../index'

describe('<DummyPage>', () => {
  it('should render and match the snapshot', () => {
    const { container } = render(<DummyPage />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
