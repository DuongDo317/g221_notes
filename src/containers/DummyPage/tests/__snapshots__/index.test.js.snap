// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DummyPage> should render and match the snapshot 1`] = `
<div
  class="sc-bwzfXH jvOKKu"
>
  <p
    class="sc-bdVaJa cEOfIf"
  >
    Foo...
     clicked
    <span
      class="counter"
    >
      0
    </span>
     times
  </p>
</div>
`;
